webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var webpack = require('webpack');
+
+module.exports = {
+	debug: true,
+	devtool: '#inline-source-map',
+	resolve: {
+    	extensions: ['', '.webpack.js', '.web.js', '.js']
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7091543/24642665/9328f02c-1911-11e7-8ae7-c1efc0decb5a.png)

In my editor I see the `indent space`. Have any clues regards this?